client/css/style.css

```diff
@@ -1038,11 +1038,11 @@ kbd {
 }
 
 #chat .self .text {
-	color: #999;
+	color: #444;
```

Comment:
Before | After
--- | ---
<img width="395" alt="screen shot 2017-08-12 at 21 21 42" src="https://user-images.githubusercontent.com/113730/29245328-0edb3ff0-7fa5-11e7-8fb5-140fcb092f11.png"> | <img width="391" alt="screen shot 2017-08-12 at 21 21 51" src="https://user-images.githubusercontent.com/113730/29245329-0edc8ea0-7fa5-11e7-860d-01ee36419d89.png">

I honestly can't tell the difference between colors in the second screenshot :/
I understand `#999` isn't contrasted enough, and this is a pretty important message to read, so maybe `#888` or `#777` instead? (second one passes AA levels)